src/main/templates/net/imagej/ops/Ops.list

```diff
@@ -104,7 +104,7 @@ namespaces = ```
 		[name: "paddingIntervalOrigin",          iface: "PaddingIntervalOrigin"],
 		[name: "sigma",                          iface: "Sigma",               aliases: ["sigmaFilter", "filterSigma"]],
 		[name: "variance",                       iface: "Variance",            aliases: ["varianceFilter", "filterVariance", "var", "varFilter", "filterVar"]],
-		[name: "vesselness",			 iface: "Vesselness"],
+		[name: "frangiVesselness",			 iface: "FrangiVesselness"],
```

Comment:
Be careful with the word [refactoring](https://en.wikipedia.org/wiki/Code_refactoring)—it means to change the internal structure of code _without_ changing its external behavior. So no API changes! In this case, the API definitely does change because the op interface name changed from `Vesselness` to `FrangiVesselness`. So this is _not_ a refactoring. A better commit message would be "Rename vesselness to frangiVesselness."

Reply:
Fixed!